### PR TITLE
!botrole failing on new guild

### LINKED ISSUE #114: !botrole breaks when joining a new server
**Describe the bug**
!botrole will break the bot because it can not return a NoneType.

**To Reproduce**
1. Let Lizard-BOT join a new server
2. Use !botrole command
3. See error message

**Expected behavior**
!botrole should either return "No role" or "Everyone"

**Screenshots**
I am an idiot and deleted the message, but it occured in Mr_Madgear tourney's server.
image

**Additional context**
I broke it 😢

### PR DESCRIPTION
**Pull request by nogarremi**
Closes #114 
Updated what the default value for `disabled_list` should be
Added additional checking to `!botrole`
`-Nog`

### CODE REVIEW

File: bot/commands/commands.py [hunk 1]
```diff
@@ -12,8 +12,14 @@
 @register('botrole')
 @register('role')
 async def botrole(command, msg, user, channel, *args, **kwargs):
+    botrole = await read_db('guild', 'botrole', kwargs['guild'])
+    try:
+        botrole_name = channel.guild.get_role(botrole).name
+    except:
+        botrole_name = '@everyone'
```
Comment (lizardman301) [inline bot/commands/commands.py:19]: should be `everyone` not `@everyone`. Right now all other names will have no @ so it doesn't accidentally ping the role

**Review by lizardman301 — COMMENTED** [review]
Small change on botrole_name to make it not use the actual `@everyone` role and to make it more consistent with the other botrole_names
image

File: bot/commands/commands.py (hunk 1 above)
Comment (nogarremi) [reply bot/commands/commands.py]: Solved in newest commit

**Review by lizardman301 — APPROVED** [review]
👍

**Pull request merged**